config/initializers/raven.rb

```diff
@@ -1,11 +0,0 @@
-# frozen_string_literal: true
-
-require 'raven'
-
-if Rails.env.eql?('production') && ENV['SENTRY_DSN'].present?
-  Raven.configure do |config|
-    config.dsn = ENV['SENTRY_DSN']
-    config.environments = Rails.env
-    config.sanitize_fields = Rails.application.config.filter_parameters.map(&:to_s)
```

Comment:
This ensures that certain fields (eg password) do not appear in the fields. Is it part of the default configuration for `sentry-rails`?

Reply:
According to this https://github.com/getsentry/sentry-ruby sentry-ruby does not report personably identifiable information (including the request body) by default. I believe this would stop passwords from appearing? config.sanitize_fields was removed (https://docs.sentry.io/platforms/ruby/migration/)